js/igv-create.js

```diff
@@ -497,6 +501,80 @@ var igv = (function (igv) {
 
     }
 
+    igv.displayChromosomePanel = function () {
+     var canvas = document.createElement('canvas');
+     canvas.id     = "scaleDisplay";
+     canvas.className = 'igv-chromosome-display-panel';
+     canvas.style.width  = document.body.clientWidth;
+     canvas.style.height = document.body.clientHeight/3;
+     canvas.style.position = "relative";
+     canvas.style.display = 'block';
+     canvas.style.overflow = 'auto';
+     canvas.style.float = 'left';
+     document.getElementById('igvControlDiv').appendChild(canvas);
+     igv.resizeDisplay();
+   }
+
+   igv.resizeDisplay = function () {
+     var canvas=document.getElementById('scaleDisplay');
+     canvas.width  = document.body.clientWidth;
+     canvas.height = document.body.clientHeight/3;
```

Comment:
why do you need to assign canvas size again when you already did in displayChromosomePanel()?